src/main/scala/beam/agentsim/agents/ridehail/RideHailModifyPassengerScheduleManager.scala

```diff
@@ -148,19 +148,8 @@ class RideHailModifyPassengerScheduleManager(
         assert(vehicleId == modifyPassengerScheduleStatus.vehicleId)
         assert(tick == modifyPassengerScheduleStatus.tick)
 
-//        log.debug(
-//          "RideHailModifyPassengerScheduleManager.handleInterrupt: " + modifyPassengerScheduleStatus.toString
-//        )
-        var reservationModifyPassengerScheduleStatus =
-          mutable.ListBuffer[RideHailModifyPassengerScheduleStatus]()
-
-        for (modifyPassengerScheduleStatus <- getWithVehicleIds(
-               modifyPassengerScheduleStatus.vehicleId
-             )) {
-          if (modifyPassengerScheduleStatus.interruptOrigin == InterruptOrigin.RESERVATION) {
-            reservationModifyPassengerScheduleStatus += modifyPassengerScheduleStatus
-          }
-        }
+        val reservationModifyPassengerScheduleStatus = getWithVehicleIds(modifyPassengerScheduleStatus.vehicleId)
```

Comment:
It's worth noting that we should be cautious of these changes. As this will result in additional allocations and GC hits depending on the size of the list. Maybe we should always specifically comment when we make use of mutable buffers for performance reasons? @REASY FYI